src/display.rs

```diff
@@ -389,6 +399,17 @@ fn get_pretty_size(node: &DisplayNode, is_biggest: bool, display_data: &DisplayD
     }
 }
 
+fn get_pretty_file_modified_time(timestamp: i64) -> String {
+    // println!("{}", timestamp);
+    let datetime: DateTime<Utc> = Utc.timestamp_opt(timestamp, 0).unwrap();
+
+    // 根据本地时区进行转换
+    let local_datetime = datetime.with_timezone(&Local);
+
+    // 按照指定格式格式化为字符串
+    local_datetime.format("%Y%m%dT%H:%M:%S").to_string()
+}
```

Comment:
Maybe: "%Y-%m-%dT%H:%M:%S"

Also lets get rid of the comments.

Reply:
I will resubmit the code after testing the performance and fix this together

Reply:
Yeah, you are right! I will conduct performance testing on my current code submission, modify the time format, and remove the unnecessary comments before resubmitting the code.